internal/pkg/cli/task_run.go

```diff
@@ -438,6 +438,10 @@ func (o *runTaskOpts) runTask() ([]*task.Task, error) {
 	o.spinner.Start(fmt.Sprintf("Waiting for %s to be running for %s.", english.Plural(o.count, "task", ""), o.groupName))
 	tasks, err := o.runner.Run()
 	if err != nil {
+		if err == task.ErrNoSubnetFound {
```

Comment:
nit: so that wrapped errors are also matched :)
```suggestion
		if errors.Is(er, task.ErrNoSubnetFound) {
```